stable/fluentd/templates/deployment.yaml

```diff
@@ -32,6 +32,9 @@ spec:
       labels:
         app: {{ template "fluentd.name" . }}
         release: {{ .Release.Name }}
+{{- with .Values.podLabels }}
```

Comment:
could you rename this property to be deployment.labels analogously to the ingresses.labels property ?

Reply:
The deployment.labels could be reused for the pod labels and the deployment resource itself.

Reply:
Done.